projects/kit/src/lib/masks/date-range/date-range-mask.ts

```diff
@@ -34,34 +36,34 @@ export function maskitoDateRangeOptionsGenerator({
 
     return {
         ...MASKITO_DEFAULT_OPTIONS,
-        mask: [...dateMask, ...Array.from(DATE_RANGE_SEPARATOR), ...dateMask],
+        mask: [...dateMask, ...Array.from(rangeSeparator), ...dateMask],
         overwriteMode: 'replace',
         preprocessors: [
             createZeroPlaceholdersPreprocessor(),
             createValidDatePreprocessor({
                 dateModeTemplate,
                 dateSegmentsSeparator: separator,
-                datesSeparator: DATE_RANGE_SEPARATOR,
+                datesSeparator: rangeSeparator,
```

Comment:
Maybe call it `datesSeparator` so the name matches and can be written just as `datesSeparator,`? Otherwise it's 2 terms that mean the same thing.

Reply:
good point, i will fix)

Reply:
At present time, [DateRange](https://tinkoff.github.io/maskito/kit/date-range/API) already has property `[separator]` (separator between date segments).
If we call new property as `datesSeparator`, then it can become too difficult to distinguish them

```ts
import {maskitoDateRangeOptionsGenerator} from '@maskito/kit';

const options: MaskitoOptions = maskitoDateRangeOptionsGenerator({
    separator: '/',
    datesSeparator: ' ~ ',
});
```

___

## Proposed new public API

I propose:

1. To deprecate old property `[separator]` (and call it as `dateSegmentsSeparator`).
2. Call new property as `rangeSeparator` (and rename `datesSeparator` inside internal utilities).

So, the final version of API:
```ts
import {maskitoDateRangeOptionsGenerator} from '@maskito/kit';

// 1
const options: MaskitoOptions = maskitoDateRangeOptionsGenerator({
    dateSegmentsSeparator: '/',
    rangeSeparator: ' ~ ',
});
```

What do you think, guys ?

__
## Other possible version of new public API

```ts
import {maskitoDateRangeOptionsGenerator} from '@maskito/kit';

// 2
const options: MaskitoOptions = maskitoDateRangeOptionsGenerator({
    dateSegmentsSeparator: '/',
    datesSeparator: ' ~ ',
});

// 3
const options: MaskitoOptions = maskitoDateRangeOptionsGenerator({
    dateSegmentsSeparator: '/',
    dateRangeSeparator: ' ~ ',
});
```

Reply:
I like 1st option, @waterplea that do you think ?

Reply:
@WoodenPC We've discussed this issue with the team, and decided to choose this option:

```ts
const options: MaskitoOptions = maskitoDateRangeOptionsGenerator({
    dateSeparator: '/',
    rangeSeparator: ' ~ ',
});
```

It means that you can keep naming of new property `rangeSeparator` untouched.
In next PR we can deprecate the property `separator` (from `DateRange`) and introduce new `dateSeparator`.
You can do it too (if your wish, of course) or I will do it by myself later.

Reply:
@nsbarsukov lets keep it as is for now and deprecate `separator` in future PR. I saved my changes without any inner args renaming.  Psz resolve conversation if no more changes needed in this part of code)